tests/Model/Generated/Endpoint/TabUsageSingleTest.php

```diff
@@ -32,44 +33,24 @@ class TabUsageSingleTest extends BunqSdkTestBase
      */
     const AMOUNT_IN_EUR = '0.02';
 
-    /**
-     * @var int
-     */
-    private static $cashRegisterId;
-
     /**
      * @var string
      */
     private static $tabUuid;
 
-    /**
-     */
-    public static function setUpBeforeClass()
-    {
-        parent::setUpBeforeClass();
-        static::$cashRegisterId = Config::getCashRegisterId();
-    }
-
-    /**
-     * Deletes the created tab.
-     */
-    public static function tearDownAfterClass()
-    {
-        TabUsageSingle::delete(
-            static::$cashRegisterId,
-            static::$tabUuid
-        );
-    }
-
     /**
      * Tests the creation of a tab.
      *
      * This test and all its dependants have no assertion as of its testing to see if the code runs without errors.
      */
     public function testCreateTab()
     {
+        if (BunqContext::getUserContext()->isOnlyUserPersonSet()) {
+            static::markTestSkipped('Only user company can create tabs.');
+        }
```

Comment:
explicit else

Reply:
In open-source project explicit else is not needed. Only in bunq internal projects.

Reply:
Ack, man, this switching between style guides... 😅